app/main.py

```diff
@@ -1 +1,10 @@
-# write your code here
+def copy_file(command: str) -> None:
+    if (
+            len(command.split()) == 3
+            and command.split()[0] == "cp"
+            and command.split()[1] != command.split()[2]
+    ):
+
+        with (open(command.split()[1], "r") as file,
+              open(command.split()[2], "w") as new_file):
+            new_file.write(file.read())
```

Comment:
Make the `split` operation once and assign output to a variable afterward. Check the length of the obtained variable and in case it equals 3 perform unpacking and use newly created variables during two checks that are left. Avoid using indexing